.github/workflows/binaries.yml

```diff
@@ -0,0 +1,460 @@
+name: Release extension with new runme binaries
```

Comment:
if this is a release workflow maybe call the file `release.yml`?

Reply:
yes, the idea is to eventually call this file release.yml once we are 100% sure everything works and remove the current release.yml file.